pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -278,6 +280,7 @@ func (a *Bootstrap) getTemplateData(installConfig *types.InstallConfig, releaseI
 		BootImage:             string(*rhcosImage),
 		PlatformData:          platformData,
 		ClusterProfile:        clusterProfile,
+		BootstrapInPlace:      installConfig.BootstrapInPlace,
```

Comment:
This makes it so that if the user includes the `bootstrapInPlace` field in the install config, then the bootstrap ignition created be calling `create ignition-configs` or `create cluster` will not work. The `bootkube.sh` will be set up for bootstrap-in-place.

Reply:
That is correct.
To be honest I think that once we added the bootstrapInPlace field to the install config there is no longer reason to have the `create single-node-ignition-config` command.
I'll send a PR to the enhancement proposing that change.
How do you suggest to mitigate that?, seems that I should add the BootstrapInPlace bool back to `Bootstrap`